Notifier desktop app

TanCodes opened the pull request:
This is an desktop notifier app build using python.

prathimacode-hub reviewed (changes requested):
Kindly follow the project structure given.
Add PR template along with issue number. 

@TanCodes

TanCodes commented:
## Related Issue 

- This is an desktop notifier app build using python. Also added PR template along with issue number.

- [x] Hacktoberfest2021 Participant 
- [x] Contributor

Closes: #1020

### Describe the changes you've made

Give a clear description what modifications you have made

## Type of change

What sort of change have you made:
<!--
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

It has been tested with various tests and conditions.

## Checklist:
<!--
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.
